packages/ui/packages/Input/src/index.js

```diff
@@ -21,6 +21,7 @@ const Input = React.forwardRef((props, ref) => {
       )}
       <StyledInput
         data-testid="input"
+        autocomplete="off"
```

Comment:
I think it is better to have autocomplete on, and make it off it any case in components which we want [ex: in search bar]